docs/flatcar-linux/packet.md

```diff
@@ -269,6 +270,7 @@ Check the [variables.tf](https://github.com/kinvolk/lokomotive-kubernetes/blob/m
 | facility | Packet Region in which the instance(s) should be deployed | https://www.packet.com/developers/api/#facilities. Eg: "ams1" |
 | pool_name | Name of the worker pool. Used in setting hostname | "helium" |
 | kubeconfig | Kubeconfig to be used in worker pools | "${module.controller.kubeconfig} |
+| worker_clc_snippets | Worker Container Linux Config snippets | [] | [example](../advanced/customization.md#usage) |
```

Comment:
Rename to `clc_snippets`

Reply:
I think that should be the name *for the worker pool module* (packet/flatcar-linux/kubernetes/workers)


On the top level module (packet/flatcar-linux/kubernetes) it should be with the workers prefix, but we don't support that for packet (seems like a bug to me) and these docs are about the worker module. So, dropping the worker prefix here is the right thing, I think :)